src/util/HighsHashTree.h

```diff
@@ -655,6 +655,9 @@ class HighsHashTree {
               for (int i = 0; i <= newNumChild; ++i)
                 mergeIntoLeaf(newLeafSize4, hashPos, branch->child[i]);
             }
+            default:
+              throw std::logic_error(
+                  "Unexpected result from entries_to_size_class");
```

Comment:
Note the discussion on exceptions here: https://github.com/ERGO-Code/HiGHS/discussions/1368

The basic takeaway is that since they require catching and handling before being passed to the C API they might not be of much use without a more principled effort.

Reply:
At the very least it should be caught and rethrown to ensure correct propagation.